src/main/java/uk/gov/hmcts/darts/audio/component/impl/AudioRequestBeingProcessedFromArchiveQueryImpl.java

```diff
@@ -17,14 +17,13 @@
 @Component
 @RequiredArgsConstructor
 public class AudioRequestBeingProcessedFromArchiveQueryImpl implements AudioRequestBeingProcessedFromArchiveQuery {
-
     private final NamedParameterJdbcTemplate jdbcTemplate;
     private final AudioRequestBeingProcessedFromArchiveQueryResultRowMapper rowMapper;
 
     @Override
     public List<AudioRequestBeingProcessedFromArchiveQueryResult> getResults(Integer mediaRequestId) {
         return jdbcTemplate.query(
-            """
+                """
```

Comment:
Can you restore the white space in this file - not sure if it makes any difference here, but notice your new Query class has it like I had it originally?